packages/oauth-providers/package.json

```diff
@@ -96,6 +109,10 @@
     "tsup": "^8.0.0",
     "typescript": "^5.2.2"
   },
+  "dependencies": {
+    "@types/crypto-js": "^4.2.1",
```

Comment:
I think we can remove these two `crypto-js` dependencies. Right?

Reply:
My bad, I thought I removed them already